### PR TITLE
Load token from command

### PR DESCRIPTION
**Pull request by sonjaer**
# TL;DR

Let users configure to load the id token from an external process.
In our case, we would run `gcloud auth print-identity-token`

```
[credentials]
auth_mode=external_process
command=gcloud,auth,print-identity-token
```

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by sonjaer** [comment]
hmm, why doesn't this build kick off?

**Review by EngHabu — APPROVED** [review]
Thank you!

**Comment by wild-endeavor** [comment]
or just leave as is...
